extensions/gitpod-web/src/extension.ts

```diff
@@ -577,6 +577,27 @@ function registerPorts(context: GitpodExtensionContext): void {
 			}
 		});
 	}
+
+	const packageJSON = context.extension.packageJSON;
+	const experiments = new ExperimentalSettings(packageJSON.configcatKey, packageJSON.version, context.logger);
+	context.subscriptions.push(experiments);
+
+	const isSaaSGitpod = context.info.getGitpodHost() === 'https://gitpod.io';
+	async function getPortsViewExperimentEnable(): Promise<boolean> {
+		const isEnabled = isSaaSGitpod
+			? (await experiments.get<boolean>('gitpod.experimental.portsView.enabled', (await context.user).id))!
+			: vscode.workspace.getConfiguration('gitpod').get<boolean>('experimental.portsView.enabled')!;
```

Comment:
Should this apply to all users or not @akosyakov ?

Reply:
I think generally there is no big concern, since SH installations should be updated to have this code. Hopefully we finish experiment but next SH release. But just to be on safe side let's have a guard. Imagine if SH team does a patch release and enables an experiment for SH users or something like that.